AUTHORS.md

```diff
@@ -98,6 +99,12 @@ possible:
  * yilinwei
  * Zach Abbott
 
+Cats has been heavily inspired by [scalaz](https://github.com/scalaz/scalaz),
+and some Cats code is only a slightly modified version of code originating in
+scalaz. Therefore, we'd also like to credit and thank all of the
```

Comment:
👍 thanks good catch. I tend to capitalize Cats to make it clear that I'm talking about the project and not the group of animals, but I definitely didn't mean to be inconsistent here.